tests/app/Infrastructure/Http/Requests/OpenWalletRequestTest.php

```diff
@@ -1,8 +1,8 @@
 <?php
 
-namespace Tests\app\Http\Requests;
+namespace Tests\app\Infrastructure\Http\Requests;
```

Comment:
In addition to sorting the tests by the layers of the hexagonal architecture, we also sort them by the type of test: Unitary, integration or e2e.

Reply:
By this do you mean that we should create a folder for unit tests and another for integration tests?